openmoc/plotter.py

```diff
@@ -37,6 +37,11 @@
 # in your ~/.matplotlib/matplotlibrc
 plt.ioff()
 
+## Font parameters to use in all plots
```

Comment:
That is there since that is doxygen syntax for global python variables. I'd prefer to keep consistent with doxygen syntax for our comments until (unless) we make a systematic effort to transition towards only docstrings only in our Python code.